CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## Unreleased
+
+> ![Important]
+> Version 8.21.0 introduced an issue for macOS apps that run without a sandbox (i.e. distributed outside the Mac App Store). We recommend upgrading to at least this version.
```

Comment:
`m`: I think it's essential to point out what the issue is. Otherwise, people will wonder what issue?? Same applies for 8.21.0 please.
```suggestion
> Version 8.21.0 introduced an issue for app launch profiling **only for macOS apps that run without a sandbox** (i.e. distributed outside the Mac App Store). This issue could lead to starting the app launch profiler even when it's not configured via the options. We recommend upgrading to at least this version.
```